jquery.orgchart.js

```diff
@@ -28,7 +28,14 @@
             // Store the original element
             $this.data("chart-source", $chartSource);
             // Build the chart...
-            var $container = $("<div class='" + opts.chartClass + "'/>");
+            var chart_cls = '';
+            if(opts.direction == 'rtl'){
+                chart_cls = opts.chartClass+' rtl';
+            }else{
+                chart_cls = opts.chartClass+' ltr';
+            }
```

Comment:
Ternary operator would be better here, in one line.
chart_cls is not a conventional variable name, chartClass would be better.